src/main/java/cc/polyfrost/oneconfig/config/Config.java

```diff
@@ -193,17 +213,18 @@ protected final void generateOptionList(Object instance, OptionPage page, Mod mo
                 else subcategory.bottomButtons.add(button);
             } else if (field.isAnnotationPresent(HUD.class)) {
                 HUDUtils.addHudOptions(page, field, instance, this);
+            } else if (isHypixelKey) {
+                throw new IllegalStateException("Field " + field.getName() + " is missing @Text annotation! This is required for Hypixel keys!");
             }
         }
-        /*for (Method method : instance.getClass().getDeclaredMethods()) {
+        for (Method method : instance.getClass().getDeclaredMethods()) {
             Button button = ConfigUtils.findAnnotation(method, Button.class);
             String optionName = pagePath + method.getName();
             if (button != null) {
-                ConfigButton option = ConfigButton.create(method, instance);
-                ConfigUtils.getSubCategory(page, button.category(), button.subcategory()).options.add(option);
+                BasicOption option = ConfigUtils.addOptionToPage(page, method, instance);
                 optionNames.put(optionName, option);
             }
-        }*/
+        }
```

Comment:
can't wait for all buttons using methods to just be at the bottom

Reply:
> can't wait for all buttons using methods to just be at the bottom

I fixed it